internal/namespaces/k8s/v1beta4/custom_node.go

```diff
@@ -0,0 +1,57 @@
+package k8s
+
+import (
+	"context"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-cli/internal/human"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	nodeActionTimeout = 10 * time.Minute
+)
+
+//
+// Marshalers
+//
+
+// nodeStatusMarshalerFunc marshals a k8s.ClusterStatus.
+var (
+	nodeStatusAttributes = human.Attributes{
+		k8s.NodeStatusCreating:      color.FgBlue,
+		k8s.NodeStatusRebooting:     color.FgBlue,
+		k8s.NodeStatusReady:         color.FgGreen,
+		k8s.NodeStatusNotready:      color.FgYellow,
+		k8s.NodeStatusCreationError: color.FgRed,
+		k8s.NodeStatusLocked:        color.FgRed,
+		k8s.NodeStatusWarning:       color.FgHiYellow,
+	}
+)
+
+const (
+	nodeActionReboot = iota
```

Comment:
will there be other actions in the future ?

Reply:
there could be :sweat_smile: